### PR TITLE
Upgrade supported storage database version in the CI

### PR DESCRIPTION
**Pull request by thongdk8**
Please take a look! Thank you!

### CODE REVIEW

**Comment by thongdk8** [comment]
@brfrn169 The CI with the new version of the Postgres database is failed in the multiple clustering keys integration test. Could you take a look at it?

**Comment by thongdk8** [comment]
@brfrn169 It is a docker image issue I think, I changed the image to the official Postgres one instead of from the circleci and now it is fine.

File: .circleci/config.yml [hunk 1]
```diff
@@ -251,7 +251,7 @@ jobs:
   integration-test-for-jdbc-postgresql:
     docker:
       - image: circleci/openjdk:8-jdk
-      - image: circleci/postgres:9.6.9-alpine
+      - image: postgres:9.6.24-alpine
```
Comment (brfrn169) [inline .circleci/config.yml:254]: Can you please upgrade to PostgreSQL 12. It's the latest version we support:
https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md

**Review by brfrn169 — COMMENTED** [review]
Left a comment. Please take a look!

Also, please upgrade mysql for multi-storage integration test as well. And can you please upgrade Oracle to 19? And SQL Server to the latest version?

Note that, we need to build an Oracle express edition docker image and push it to our private repository and use it.

File: .circleci/config.yml (hunk 1 above)
Comment (feeblefakie) [reply .circleci/config.yml:254]: Just a note. I benchmarked Scalar DL on PostgreSQL 14 (the latest stable) with TPCC and nothing went wrong.
So, maybe it's good enough to say 14. Of course, we can (or should) wait for the QA team verification.

Comment (brfrn169) [reply .circleci/config.yml:254]: @thongdk8 So let's use PostgreSQL 14.

Comment (thongdk8) [reply .circleci/config.yml]: I updated the version of the databases. Please take a look again. Thank you.

**Review by brfrn169 — COMMENTED** [review]
@thongdk8 Can you please upgrade SQL Server version to the latest version as well?

And please don't do force push after review starts?

**Comment by thongdk8** [comment]
> @thongdk8 Can you please upgrade SQL Server version to the latest version as well?
> 
> And please don't do force push after review starts?

@brfrn169 We are using the latest version of SQL Server I think. Sorry, I'll not do a force push futher

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**